<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>360手机助手登陆</string>
   </property>
  </widget>
  <widget class="QWidget" name="topWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>97</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pbClose">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lUserName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPsw">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePsw">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRegister">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFindPsw">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>97</y>
     <width>366</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="cbRememberPsw">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>记住账号</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAutoLogin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>自动登陆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>登陆</string>
    </property>
   </widget>
  </widget>
  <zorder>mainWidget</zorder>
  <zorder>topWidget</zorder>
  <zorder>lbTitle</zorder>
  <zorder>lUserName</zorder>
  <zorder>lPsw</zorder>
  <zorder>leUserName</zorder>
  <zorder>lePsw</zorder>
  <zorder>pbRegister</zorder>
  <zorder>pbFindPsw</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
